doc/examples/example_uncertainty.py

```diff
@@ -27,14 +27,14 @@
 print('percentage inside:', np.count_nonzero(inside) / len(inside) * 100)
 
 # # Plot some results
-axes = ml.plots.results(tmin="2010", tmax="2015", figsize=(10, 6))
-axes[0].fill_between(df.index, df.iloc[:, 0], df.iloc[:, 1], color="gray",
-                     zorder=-1, alpha=0.5, label="95% Prediction interval")
-axes[0].legend(ncol=3)
+axs = ml.plots.results(tmin="2010", tmax="2015", figsize=(10, 6))
```

Comment:
what did the 'e' do wrong ... :P?

Reply:
Nothing, this is me trying to clean up the names since the type name is now `Axes` but we in a lot of places in the code we assign the variable name `axes` to a list or dict of `Axes`. However, I gave up renaming because we use the name `axes` too much, also as an attribute name e.g. in the CompareModels class.